### DISCUSSION TITLE
Why in Andes, Pmax multiplied by the system base value is different from Sn in Andes case input files?

### DISCUSSION THREAD

**Discussion by sghqsads** [CURENT/andes#479] [Q&A] [answered]
Hello everyone,

I was wondering why Pmax multiplied by the system base value is different from Sn in Andes case input files?

For instance, in ieee 39_full.xlsx, 

Sn:
1040
836
843.7
1174.8
1080.2
1085.7
1025.2
970.2
1684.1

pmax:
10.4
6.5
7.5
6.82
5.5
6.97
5.95
5.64
8.65

What is the reason for this setting? Thank you！

**Comment by jinningwang** [comment, marked as answer]
I assume you are referring to the sheet `PV`. In it, the parameter ``pmax`` is tuned results by me.

The reason is that I used this file to do the OPF study later on, where ``pmax`` will be converted to pmax of the generator in pandapower. It has a significant impact on the economic results, whereas it seldom impacts the dynamic simulation in ANDES.

**Reply by sghqsads** [reply]
Thank you!
